mrp_bom_location/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "MRP BOM Location",
+    "summary": "Adds location field to Bill of Materials and its components.",
+    "version": "13.0.1.0.1",
+    "category": "Manufacture",
+    "website": "https://github.com/OCA/manufacture",
+    "author": "Eficent, Odoo Community Association (OCA)",
```

Comment:
Could you change Eficent by ForgeFlow? Thank you!

Reply:
I can but it is really a valid change as we will be changing the author.

Reply:
Yes, it is. Our company has been rebranded: https://www.forgeflow.com/blog/our-blog-1/post/from-eficent-to-forgeflow-2

Reply:
But I think we can merge like this and we can open a new PR to change the copyrights, so no worries.